browser/backend/code/common/authorizer.py

```diff
@@ -8,7 +8,7 @@
 USERINFO_ENDPOINT = "https://czi-single-cell.auth0.com/userinfo"
 
 AUTH0_DOMAIN = "czi-single-cell.auth0.com"
-API_AUDIENCE = f"https://browser-api.{os.environ['DEPLOYMENT_STAGE']}.single-cell.czi.technology"
+API_AUDIENCE = f"https://api.corpora.{os.environ['DEPLOYMENT_STAGE']}.single-cell.czi.technology"
```

Comment:
Do we have plans to use the domain `corpora.cziscience.com`?

Reply:
Yep, it should be the one that Calvin mentioned.

Reply:
What is the URL for the site, and what is the URL for the api?

Reply:
corpora.cziscience.com
api.corpora.cziscience.com

Reply:
These domains are for productions only right? It looks like cellxgene is using cellxgene.staging.single-cell.czi.technology for is development stage. Do we want to follow that same naming scheme as cellxgene for our development environements?

Reply:
You'd tag on the `staging` or `dev` environment name appropriately.

I spoke with Bruce and Madison about the cellxgene URLs and they largely used them due to convenience and bc they were free. We aren't tied to matching the same naming scheme as cellxgene so let's keep ours simple and only use the cziscience.com domain unless there's a strong reason not to.